tfctl/replan.go

```diff
@@ -36,7 +36,8 @@ func (c *CLI) Replan(out io.Writer, resource string) error {
 	}
 	fmt.Fprintf(out, " Replan requested for %s/%s\n", c.namespace, resource)
 
-	if err := wait.Poll(1*time.Second, 30*time.Second, func() (bool, error) {
+	// wait for the plan to be ready, 4 loops, 30 seconds each
+	if err := wait.Poll(1*time.Second, 120*time.Second, func() (bool, error) {
```

Comment:
`wait.Poll` is deprecated in newer versions of the `wait` package. ([see issue in weave-gitops](https://github.com/weaveworks/weave-gitops/issues/3812)).

No changes required, just noted as a reminder we have to deal it here too later.

Reply:
ack